.changeset/big-hairs-remember.md

```diff
@@ -0,0 +1,6 @@
+---
+'@shopify/cli-kit': minor
+'@shopify/theme': minor
```

Comment:
`patch`

Reply:
why `patch`? Isn't that for bug [fixes](https://semver.org/)?

Reply:
Judging off the current contents of `.changeset` it doesn't seems like to be a convention.

I see other similar changes with `patch` let's go with that for this PR.

I'll follow up with the foundations team to confirm